app/move/controllers/create/hospital-appointment.js

```diff
@@ -0,0 +1,44 @@
+const { formatISO, isValid, parseISO } = require('date-fns')
+
+const filters = require('../../../../config/nunjucks/filters')
+
+const CreateBaseController = require('./base')
+
+class HospitalAppointmentController extends CreateBaseController {
+  process(req, res, next) {
+    const startTime = req.form.values.hospital_appointment__start_time
+
+    if (startTime) {
+      const moveDate = req.getMove().date
+      const moveDateTime = parseISO(`${moveDate}T${startTime}`)
+      req.form.values.hospital_appointment__start_time = isValid(moveDateTime)
+        ? formatISO(moveDateTime)
+        : startTime
+    }
+
+    req.form.values.time_due = req.form.values.hospital_appointment__start_time
```

Comment:
I wonder if you could simplify this by creating a field called `time_due`?

The reason I used the prefix for court hearings was because they needed to be sent to a different endpoint so created a different name. It might be nice if we have a single field to send the `time_due` value to the API.

Reply:
Simplified and using `time_due` now that the field's context can be set